Userland/Applications/Browser/main.cpp

```diff
@@ -69,7 +69,17 @@ int main(int argc, char** argv)
         return 1;
     }
 
-    if (unveil("/home", "rwc") < 0) {
+    if (unveil(String::formatted("{}/BrowserContentFilters.txt", Core::StandardPaths::config_directory()).characters(), "rwc") < 0) {
```

Comment:
```suggestion
    if (unveil(LexicalPath::join(Core::StandardPaths::config_directory(), "BrowserContentFilters.txt").characters(), "rwc") < 0) {
```
Same for the two `unveil` calls below. Also, you should probably cache the `Core::StandardPaths::config_directory()` in a variable to avoid threefold StringBuilder and String allocation inside that function call.